librz/core/cbin.c

```diff
@@ -3810,8 +3810,9 @@ RZ_API bool rz_core_bin_class_methods_print(RzCore *core, RzBinFile *bf, RzCmdSt
 			break;
 		case RZ_OUTPUT_MODE_JSON:
 			rz_list_foreach (c->methods, iter2, sym) {
+				const char *name = sym->dname ? sym->dname : sym->name;
```

Comment:
please just move this outside of the switch and use it in all cases.

Reply:
done.